install/Knative-with-ICP.md

```diff
@@ -0,0 +1,185 @@
+# Knative Install on IBM Cloud Private
+
+This guide walks you through the installation of the latest version of
+[Knative Serving](https://github.com/knative/serving) using pre-built images and
+demonstrates creating and deploying an image of a sample `hello world` app onto
+the newly created Knative cluster.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires an [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster v3.1.1. See [Installing IBM Cloud Private Cloud Native, Enterprise, and Community editions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html) in the IBM Knowledge Center for install instructions.
```

Comment:
It looks like all the steps in the [KC topic](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html)  should be taken? 

If that is true, lets clarify/state that, and also provide only a single link because right now with the 6 separate links, we are asking the user to switch back and forth between the same page multiple times. 

I think you could also leave in the list but remove the links.

Reply:
@RichieEscarez  Yes, all the steps in the KC topic should be taken.  I wrote all the steps in the the previous review versions.  But it seems a little redundant. So I simplify it. 

I think I just keep the KC topic link here and remove the six separate links.  User follow the step from KC link to install the IBM Cloud Private.

Reply:
Sounds good, please revise to:

```suggestion
Knative requires a v3.1.1 standard [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster. 
Before you can install Knative, you must first complete all the steps that are provided in the 
[IBM Cloud Private standard cluster installation instructions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html).
```

Reply:
Done